src/mlpack/bindings/cli/cli_option.hpp

```diff
@@ -3,15 +3,15 @@
  * @author Matthew Amidon
  *
  * Definition of the Option class, which is used to define parameters which are
- * used by CLI.  The ProgramDoc class also resides here.
+ * used by IO.  The ProgramDoc class also resides here.
  *
  * mlpack is free software; you may redistribute it and/or modify it under the
  * terms of the 3-clause BSD license.  You should have received a copy of the
  * 3-clause BSD license along with mlpack.  If not, see
  * http://www.opensource.org/licenses/BSD-3-Clause for more information.
  */
-#ifndef MLPACK_CORE_BINDINGS_CLI_CLI_OPTION_HPP
-#define MLPACK_CORE_BINDINGS_CLI_CLI_OPTION_HPP
+#ifndef MLPACK_CORE_BINDINGS_IO_IO_OPTION_HPP
+#define MLPACK_CORE_BINDINGS_IO_IO_OPTION_HPP
```

Comment:
I think maybe a lot of the header guards will need to be fixed to use `CLI`. :+1: